pkg/client/events/websocket/dialer.go

```diff
@@ -25,9 +28,21 @@ func (wsDialer *websocketDialer) DialContext(
 	ctx context.Context,
 	urlString string,
 ) (client.Connection, error) {
+
+	dialer := websocket.DefaultDialer
+
+	if strings.HasPrefix(urlString, "wss://") {
+		systemRoots, err := x509.SystemCertPool()
+		if err != nil {
+			return nil, err
+		}
+
+		dialer.TLSClientConfig = &tls.Config{RootCAs: systemRoots}
```

Comment:
Are you sure we need to set the `RootCAs` field manually? The go documentation seems to imply that this is the default: https://pkg.go.dev/crypto/tls#Config

```suggestion
		dialer.TLSClientConfig = &tls.Config{}
```

Reply:
Good catch, done.